src/templates/chapter.js

```diff
@@ -9,14 +9,27 @@ import { Button } from '../components/button'
 
 import classes from '../styles/chapter.module.sass'
 
+import warmupOrder from '../../warmups/order'
+
 const Template = ({ data }) => {
     const { markdownRemark, site } = data
     const { courseId } = site.siteMetadata
-    const { frontmatter, htmlAst } = markdownRemark
-    const { title, description, prev, next, id } = frontmatter
+    const { frontmatter, htmlAst, fields } = markdownRemark
+    const { title, description } = frontmatter
     const [activeExc, setActiveExc] = useState(null)
-    const [completed, setCompleted] = useLocalStorage(`${courseId}-completed-${id}`, [])
+    const slug = fields.slug
+    // Use slug (which should be unique) without the starting slash to name array for tracking completed exercises
+    const [completed, setCompleted] = useLocalStorage(`${courseId}-completed-${slug.slice(1)}`, [])
     const html = renderAst(htmlAst)
+    const warmupIndex = warmupOrder.indexOf(slug)
+    if (warmupIndex < 0) {
+        console.error(`Could not find ${slug} in warmup order.
+        Please add to warmup order file to ensure that 'Previous Warmup' and 'Next Warmup' buttons behave as expected.`)
```

Comment:
Should this say "Please add it to the warmup order file ..."? That is, what about adding "it" and "the" to the sentence? Of course, I recognize that this is a very small change and only would influence what appears in the console error log.